features/networking/multus.feature

```diff
@@ -1008,3 +1008,32 @@ Feature: Multus-CNI related scenarios
     Then the output should contain:
       | 192.168.22.2     |
 
+  # @author weliang@redhat.com
+  # @case_id OCP-25915
+  @admin
+  Scenario: Multus default route overwrite
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    Given the default interface on nodes is stored in the :default_interface clipboard
+
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run oc create as admin over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/ipam-static.yaml" replacing paths:
+      | ["metadata"]["namespace"]  | <%= project.name %> | 
+      | ["metadata"]["name"]       | bridge-static       |
+      | ["spec"]["config"]| '{ "cniVersion": "0.3.0", "type": "bridge", "master": "<%= cb.default_interface %>", "ipam": {"type":"static","addresses": [{"address": "22.2.2.22/24","gateway": "22.2.2.1"}]}}' |
+    Then the step should succeed
+    
+    # Create a pod absorbing above net-attach-def
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/Pods/multus-default-route-pod.yaml |
+      | n | <%= project.name %>                                                                                                         |
+    Then the step should succeed
+    And the pod named "multus-default-route-pod" becomes ready
+
+    # Check created pod has correct default route
+    When I execute on the pod:
+      | /usr/sbin/ip | route            |
+    Then the output should contain
```

Comment:
This test will fail unless you have colon at the end of this line like ```Then the output should contain:```
Refer https://github.com/anuragthehatter/verification-tests/blob/master/features/step_definitions/common.rb#L33

Reply:
Fixed. Thanks!

Reply:
@weliang1 please run do a sanity check of your PR before submitting it by running it locally and/or against jenkins (preferable the latter) to make sure it work.

Reply:
https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3-smoke/520/consoleFull